source/deployment-guide/server/air-gapped-deployment.rst

```diff
@@ -1,200 +1,170 @@
-The process for uploading images to the registry will need to be tailored to the security policies of the air-gapped network. For example, perhaps the registry is allowed to pull images from the public internet. If not, images will have to be uploaded to the registry from a machine that has access to the public internet.
+3. **Prepare your Bill of Materials:**
 
-Ideally the air-gapped network already has a private registry available. If not, you can set up a private registry using Docker Registry or Harbor.
+Depending on your deployment method method, you'll need to download, tag, and push required materials into your private registry or mirror.  
 
-.. tab:: Docker registry
+4. **Transfer materials into the air-gapped environment:**   
 
-   1. **Install Docker Registry**:
+If the private registry cannot access the public internet, you can prepare an archive of the registry data on your internet connected machine and securely transfer it using approved data transfer methods - for example, burning to a disk.
 
-      .. code-block:: bash
+5. **Install Mattermost**
 
-         docker run -d -p 5000:5000 --restart=always --name registry registry:2
+Once you have all the necessary resources in your air-gapped environment, you can move forward with deployment following the instructions for :doc:`Linux </deployment-guide/server/deploy-linux>`, :doc:`Kubernetes </deployment-guide/server/deploy-kubernetes>`, or :doc:`Docker </deployment-guide/server/deploy-containers>`.
 
-   2. **Configure persistent storage**:
+6. **Configure Mattermost for air-gapped operation**
 
-      .. code-block:: bash
+The :ref:`configuration settings <deployment-guide/server/air-gapped-deployment:server configuration>` recommended in this document accomodate for the lack of internet access to operate Mattermost in an air-gapped environment.
 
-         docker run -d -p 5000:5000 --restart=always --name registry \
-         -v /mnt/registry:/var/lib/registry \
-         registry:2
 
-   3. **Add TLS security** (recommended):
+.. note::
+  Consider `talking to a Mattermost expert <https://mattermost.com/contact-sales/>`__ if your organization needs support deploying Mattermost and supporting services in an air-gapped environment.
 
-      a. Generate self-signed certificates:
+Bill of Materials
+-----------------
 
-         .. code-block:: bash
+On an internet connect machine, you must gather all required packages, container images, and dependencies needed for the installation process. The resources you'll need will depend on your deployment method, specifically:
 
-            mkdir -p certs
-            openssl req -newkey rsa:4096 -nodes -sha256 -keyout certs/domain.key \
-            -x509 -days 365 -out certs/domain.crt
+.. tab:: Linux
 
-      b. Run the registry with TLS:
+  Linux is recommeded as the simplest installation method for air-gapped environments. You can install the Mattermost Server in a few minutes on any air-gapped 64-bit Linux system using the tarball.
```

Comment:
"Using our provided tarball is recommeded as the simplest installation method for air-gapped environments. You can install the Mattermost Server in a few minutes on any air-gapped 64-bit Linux system."